models/_root/product.yml

```diff
@@ -1,17 +1,15 @@
 description: |-
-  Default property that provides detailed product data. Called on the default `templates/pages/product.html` and `templates/pages/amp/product.html` templates, and on several partials in the `templates/components/` subdirectory:
+  Default property that provides detailed product data. Called on the default `templates/pages/product.html` template, and on several partials in the `templates/components/` subdirectory:
+
 

```

Comment:
the following list displays as a paragraph; try making it a bulleted list & we'll see what happens.